ReactWindows/ReactNative.Net46/ReactNative.Net46.csproj

```diff
@@ -52,6 +52,10 @@
     </DefineConstants>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="AsyncClipboardService, Version=1.5.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\AsyncClipboardService.1.5.0\lib\net452\AsyncClipboardService.dll</HintPath>
```

Comment:
Seeing quite a few of these small dependencies.  Is it that much harder to use the WPF interfaces directly for clipboard?